lib/record-serialization.js

```diff
@@ -12,20 +13,23 @@ module.exports = jsrs;
 // one of our priorities to optimize it.
 const USE_NATIVE_SERIALIZER = false;
 
-const jstpNative =
-  safeRequire('../build/Release/jstp') ||
-  safeRequire('../build/Debug/jstp');
+const jstpNativeE = altConcatLazy(
```

Comment:
We have `require(name)`
We need `api.common.require(name1, ..., nameN)`
We can write `api.common.either(fn)` that which will expect `fn` is synchronous an it may raise an `Error`
If all calls to `fn` will raise, resulted function will raise too.
So we can use it like this:
```js
api.common.require = api.common.either(require);
const moduleName = api.common.require(name1, name2, name3);
```